Docs/source/manual/Model.rst

```diff
@@ -381,22 +381,39 @@ multigrid solvers. More details on how to use HYPRE is provided in control Secti
 Large Eddy Simulation
 ^^^^^^^^^^^^^^^^^^^^^
 
-To provide closure for the unresolved trubulent stress/flux terms in Large Eddy Simulation (LES), PeleLMeX supports the
-constant-coefficient Smagorinsky and WALE models for turbulent transport of momentum, species, and energy. These models are
+To provide closure for the unresolved turbulent stress/flux terms in Large Eddy Simulation (LES), PeleLMeX supports the
+constant-coefficient Smagorinsky, WALE and Sigma models for turbulent transport of momentum, species, and energy. These models are
 based on a gradient transport assumption, resulting in terms analagous to the molecular transport of these quantities, but with
-modified turbulent transport coefficients. In the Smgorinsky model, the turbulent viscosity :math:`\mu_t` is computed as:
+modified turbulent transport coefficients. The basis of all these algebraic closures is to model the subgrid scale (sgs) viscosity with:
 
 .. math::
 
-   \mu_t = \overline{\rho} C_s^2 \bar{\Delta}^2 |\widetilde{S}|, \hspace{12pt}  \widetilde{S}_{ij} = \frac{1}{2} \left(\frac{\partial u_i}{\partial x_j} + \frac{\partial u_j}{\partial x_i} \right).
+    \nu_{t} = \nu_{sgs} = ( C_m \Delta )^2 \mathcal{D}(\boldsymbol{u})
+
+where :math:`C_m` is a model constant, :math:`\Delta` is the subgrid length scale (typically the grid size) and :math:`\mathcal{D}(\boldsymbol{u})`
+is a differential operator acting on the filtered velocity. :math:`\mu_t` is then calculated using the filtered density :math:`\mu_t = \overline{\rho} \cdot \nu_{t}`.
+In the `Smgorinsky <https://doi.org/10.1175/1520-0493(1963)091<0099:GCEWTP>2.3.CO;2>`_ model, :math:`C_m = C_s = 0.18` and :math:`\mathcal{D}` writes:
```

Comment:
Fix spelling of Smagorinsky, also this link doesn't gets mangled when generating the HTML